src/utils/events/event-manager.js

```diff
@@ -1,13 +1,15 @@
-import {Manager} from 'hammerjs';
+import WheelInput from './wheel-input';
+import MoveInput from './move-input';
 
-import {
+const Manager = (typeof window === 'undefined' || typeof document === 'undefined') ?
+  ManagerMock : require('hammerjs').Manager;
```

Comment:
The mock should be there so that future consumers of EventManager can call its public API without fear of failure in some environments.